src/zope/apidoc/utilities.py

```diff
@@ -102,7 +102,7 @@ def isReferencable(path):
     # Do not allow private attributes to be accessible
     if (obj_name is not None and
         obj_name.startswith('_') and
-        not (obj_name.startswith('__') and obj_name.endswith('__'))):
+            not (obj_name.startswith('__') and obj_name.endswith('__'))):
```

Comment:
Indent the line above too, for consistency?

Reply:
Of course.